istio/updateVersion.sh

```diff
@@ -0,0 +1,15 @@
+# Derived from https://github.com/istio/istio/blob/master/install/updateVersion.sh
+
+function merge_files() {
+  ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+  ISTIO=$ROOT/istio-linkerd.yml
+
+  echo "# GENERATED FILE. Use with Kubernetes 1.5+" > $ISTIO
+  echo "# TO UPDATE, modify files in istio/ and run ./updateVersion.sh" >> $ISTIO
+  cat $ROOT/mixer-pilot.yml >> $ISTIO
```

Comment:
Not a blocker but might be nice (as a future improvement) to pull the mixer and pilot configs from the istio repo instead of copying them.